larky/src/main/resources/stdlib/xmllib.star

```diff
@@ -649,7 +649,7 @@ def XMLParser(**kw):
         if self.__seen_starttag and not self.stack:
             self.syntax_error('multiple elements on top level')
         k, j = tag.span('attrs')
-        attrdict, nsdict, k = self.parse_attributes(tagname, k, j)
+        attrdict, nsdict, k = self.parse_attributes(tagname, i+k, i+j)
```

Comment:
Can you help me understand why this change was needed?

Reply:
Because k, j are the current tag's attrib indices, i is the current tag index in the whole xml string, if not plus i, it will keep getting attrib between k and j in the first tag, not the current tag

Reply:
Ah! Thank you. I think we should put that exact comment above that line so if we ever revisit we don't have to come to the pull request to find out why it's adding i. This is great, thank you!